src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig

```diff
@@ -0,0 +1,178 @@
+                    <li>{{ 'Note that you can have several products with the same reference.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+                </ul>
+            </div>
+
+            <hr>
+
+            <div class="form-group hidden-xs-up">
+                {{ form_errors(importForm.csv) }}
+                {{ form_widget(importForm.csv, {'attr': {'class': 'js-import-file-input'}}) }}
+            </div>
+
+            <div class="form-group js-file-upload-form-group">
+                <label class="form-control-label">{{ 'Select a file to import'|trans }}</label>
+                {{ form_errors(importForm.file) }}
+                {{ form_widget(importForm.file, {'attr': {'class': 'js-import-file', 'data-max-file-upload-size': max_file_upload_size }}) }}
+                <span>{{ 'or'|trans({}, 'messages') }}</span>
+                <button type="button"
+                        class="btn btn-outline-primary btn-sm js-from-files-history-btn"
+                        {% if import_file_names is empty %}disabled{% endif %}
+                >
+                    <span class="badge badge-primary js-files-history-number">{{ import_file_names|length }}</span>
+                    {{ 'Choose from history / FTP'|trans({}, 'Admin.Advparameters.Feature') }}
+                </button>
+                <small class="form-text">{{ 'Allowed formats: .csv, .xls, .xlsx, .xlst, .ods, .ots'|trans({}, 'Admin.Advparameters.Help') }}</small>
+                <small class="form-text">{{ 'Only UTF-8 and ISO 8859-1 encodings are allowed'|trans({}, 'Admin.Advparameters.Help') }}</small>
+                <small class="form-text">{{ 'You can also upload your file via FTP to the following directory: %s .'|trans({'%s': import_dir}, 'Admin.Advparameters.Help') }}</small>
+            </div>
+
+            <div class="alert alert-danger d-none js-import-file-error" role="alert">
+                <p class="alert-text">
+                    <strong class="js-file-data"></strong>: <span class="js-error-message"></span>
+                </p>
+            </div>
+
+            <div class="alert alert-success d-none js-import-file-alert" role="alert">
+                <button type="button"
+                        class="close btn btn-outline-secondary js-change-import-file-btn"
+                        aria-label="{{ 'Change'|trans({}, 'Admin.Actions') }}"
+                >
+                    <span aria-hidden="true"><i class="material-icons">edit</i></span>
+                </button>
+                <p class="alert-text js-import-file"></p>
+            </div>
+
+            {% block import_file_history_block %}
+                {% include '@PrestaShop/Admin/Configure/AdvancedParameters/Blocks/import_file_history.html.twig' %}
+            {% endblock %}
+
+            <hr>
+
+            <div class="form-group">
+                {{ ps.label_with_help(('Language of the file'|trans), ('The locale must be installed'|trans({}, 'messages'))) }}
```

Comment:
Same here, it's better if it does not fall into the messages domain.

Reply:
Admin.Advparameters.Notifications ? It is a notification, right ?

Reply:
if we change domain, we will lose translation, is it okay to do so?

Reply:
Okay, I've just added this string to Admin.Advparameters.Notification. And the translations have been kept. So I'll do the same for the 1.7.2 branch, and then hide it from the messages domain. Is it okay for you, @sarjon and @mickaelandrieu ?

Reply:
@LouiseBonnard shouldn't it be Admin.Advparameters.Help? This domain is used in other help messages as well.

Reply:
@mickaelandrieu, what do you think ? It's hard for me to visualize the code sharply but a string containing a wording like "the XXX must be XXX" often falls into the Notification domain. But it you both think that it's better to have it into the Help domain, let me know. I will cancel my modification on the Notification .xlf file in Crowdin and add it to the Help .xlf file instead.